pkg/metrics/metrics.go

```diff
@@ -32,19 +32,25 @@ func StartMetrics(enableApplicationController, enableMigrationController bool) e
 			for {
 				isCRDRegistered := true
 				crdName := fmt.Sprintf("%s.%s", stork_api.ApplicationBackupResourcePlural, stork.GroupName)
-				if _, err := apiextensions.Instance().GetCRD(crdName, metav1.GetOptions{}); err != nil {
-					logrus.Errorf("failed to retrive applicationbackups crds: %v", err)
-					isCRDRegistered = false
+				if _, err := apiextensions.Instance().GetCRDV1beta1(crdName, metav1.GetOptions{}); err != nil {
```

Comment:
same here and other places

Reply:
do you mean `NotFound` error we get if we can't read apis via v1 libs -  ` the server could not find the requested resource`

Reply:
fixed